bigtable/snippets/filters/filter_snippets.py

```diff
@@ -0,0 +1,357 @@
+#!/usr/bin/env python
+
+# Copyright 2020, Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START bigtable_filters_limit_row_sample]
+# [START bigtable_filters_limit_row_regex]
+# [START bigtable_filters_limit_cells_per_col]
+# [START bigtable_filters_limit_cells_per_row]
+# [START bigtable_filters_limit_cells_per_row_offset]
+# [START bigtable_filters_limit_col_family_regex]
+# [START bigtable_filters_limit_col_qualifier_regex]
+# [START bigtable_filters_limit_col_range]
+# [START bigtable_filters_limit_value_range]
+# [START bigtable_filters_limit_value_regex]
+# [START bigtable_filters_limit_timestamp_range]
+# [START bigtable_filters_limit_block_all]
+# [START bigtable_filters_limit_pass_all]
+# [START bigtable_filters_modify_strip_value]
+# [START bigtable_filters_modify_apply_label]
+# [START bigtable_filters_composing_chain]
+# [START bigtable_filters_composing_interleave]
+# [START bigtable_filters_composing_condition]
+import datetime
+from google.cloud import bigtable
+
+# [END bigtable_filters_limit_row_sample]
+# [END bigtable_filters_limit_row_regex]
+# [END bigtable_filters_limit_cells_per_col]
+# [END bigtable_filters_limit_cells_per_row]
+# [END bigtable_filters_limit_cells_per_row_offset]
+# [END bigtable_filters_limit_col_family_regex]
+# [END bigtable_filters_limit_col_qualifier_regex]
+# [END bigtable_filters_limit_col_range]
+# [END bigtable_filters_limit_value_range]
+# [END bigtable_filters_limit_value_regex]
+# [END bigtable_filters_limit_timestamp_range]
+# [END bigtable_filters_limit_block_all]
+# [END bigtable_filters_limit_pass_all]
+# [END bigtable_filters_modify_strip_value]
+# [END bigtable_filters_modify_apply_label]
+# [END bigtable_filters_composing_chain]
+# [END bigtable_filters_composing_interleave]
+# [END bigtable_filters_composing_condition]
+from google.cloud.bigtable.row_filters import ApplyLabelFilter, \
```

Comment:
are all of these imports needed by each sample? Is there a way to separate them so it is a smaller subset?

Also, I think it is still the plan to split snippets to be separate files. Perhaps that would make this more straightforward?

Reply:
Ok, so the issue I was having is that the reading and printing functions are exactly the same, so I didn't want to have that code duplicated throughout in case any of it needed to change. If there is a way to split the imports, I can do that so each snippet only gets the necessary ones.

Reply:
If you want to punt this for a bit we can, but the end goal is to have 1 sample: 1 file. At that point the imports will split. For an example of of what I mean, you can look at the storage samples.